recipe/meta.yaml

```diff
@@ -1,8 +1,9 @@
 {% set version = "2022.02.2+485" %}
 {% set pkg_version = version.split("+")[0] %}
-{% set upstream_build_num = version.split("+")[1] | int %}
 # reset the below to 0 when bumping upstream build num.
-{% set cf_build_num = 1 %}
+{% set build = 1 %}
+{% set upstream_build_num = version.split("+")[1] | int %}
```

Comment:
What was the benefit of moving `upstream_build_num`? now the grouping for `split("+")` is gone and my comment is not true anymore (or at least ambiguous).

Reply:
I moved it because I don't know exactly what rules the bot follows. In https://github.com/conda-forge/libsharp-feedstock/pull/11/commits/0eacd9c5f4b0a73e40f49b0423fce9c787f7c79d# it updates the first instance of `{{% set build ...` but not the second. I moved `set build` to guard against the possibility that the bot would be confused by `set upstream_build_num` and try to change that instead. All of that is totally speculative, I don't really know what the bot does outside of the examples I mentioned, but this seemed safer to me.

Reply:
(I think it shouldn't affect the bot, but in any case, we can also sort down the rest of the split stuff after the bulld number with the next PR)